CHANGELOG.md

```diff
@@ -4,6 +4,79 @@
 
 # JupyterLab Changelog
 
+## v4.3 (beta)
+
+JupyterLab 4.3 includes a number of new features (described below), bug fixes, and enhancements.
+This release is compatible with extensions supporting JupyterLab 4.0.
+Extension authors are recommended to consult the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-4-2-to-4-3) which lists deprecations and changes to the public API.
+
+### Minimap
+
+The virtual scrollbar now acts as a minimap and exposes information on whether a cell:
+
+- is running/scheduled (`[*]`), idle (`[ ]`) or was already run (e.g. `[1]`)
+- was modified since it was last executed ("dirty") - orange background
+- has output contains an error (traceback) - red background
+- is markdown/code (`[ ]`/`[*]`/`[1]` for code, nothing for markdown)
+- how much text/code it contains
+
+<img alt="The minimap view" src="https://raw.githubusercontent.com/krassowski/jupyterlab/user-facing-changelog-4.3/docs/source/getting_started/changelog_assets/4.3-minimap.png" class="jp-screenshot">
+
+To enable the minimap, click on the hamburger icon (≡) which appears in the notebook's toolbar. If you do not see the hamburger icon, go to Settings → Notebook → Windowing mode and choose "full".
+
+### Fullscreen mode
+
+JupyterLab's now offers a way to toggle the full screen mode from within the application, either
+- using the `View` → `Appearance` → `Fullscreen Mode` menu entry
+- using a customizable shortcut, by default <kbd>F11</kbd> (custom shortcuts are supported in Chrome, Safari and Edge)
+
+### File browser enhancements
+
+The file browser now:
+- supports resizing the columns and remembers the column sizes after reloading JupyterLab
+- supports uploading folders by drag-and-drop
+- supports navigation with a single click (opt-in)
+- has the file filter collapsed by default (funnel icon), increasing the space available
+
+<img alt="The file browser with resize handle and filter funnel icon" src="https://raw.githubusercontent.com/krassowski/jupyterlab/user-facing-changelog-4.3/docs/source/getting_started/changelog_assets/4.3-filebrowser.png" class="jp-screenshot">
+
+### Server-side execution and collaboration
+
+The previous release enabled extensions to implement server-side execution which preserves
+outputs cells created when the browser window with JupyterLab gets closed or disconnected.
+The server-side execution builds on the same shared models API as the real-time collaboration (RTC).
+
+This release includes additional work benefiting both the server-side execution and RTC use cases:
```

Comment:
```suggestion
This release includes additional work, building on work in Lab 4.2. improving both
the server-side execution and real-time collaboration (RTC) use cases:
```

Reply:
I wanted to be explicit that RTC and server side execution are not built-in. But I later wrote the paragraph below which includes links to the extensions so this is not necessary to be so verbose indeed.

I still think that it is beneficial to explain what server-side execution means because most users will not know what benefits are behind it.

Reply:
```suggestion
JupyterLab 4.2 release enabled extensions to implement server-side execution which preserves
outputs created when the browser window with JupyterLab gets closed or disconnected.
The server-side execution builds on the same shared models API as the real-time collaboration (RTC).

This release includes additional work, improving both
the server-side execution and real-time collaboration (RTC) use cases:
```